e2e/fixtures/status.go

```diff
@@ -260,20 +260,29 @@ func (fdbCluster FdbCluster) StatusInvariantChecker(
 	)
 }
 
+// checkAvailability returns nil if the cluster is reachable or if not a quorum of the coordinators are reachable. If
```

Comment:
Same/Similar question: how can the cluster be reachable if a quorum of coordinators is not reachable?